docs/yaml/03-deployment.yaml

```diff
@@ -68,19 +60,35 @@ kind: Service
 apiVersion: v1
 metadata:
   name: gangway
-  # example annotations if using AWS ELB with an SSL CERT
-  annotations:
-    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: <aws arn for a cert>
-    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
-    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
-    service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: "true"
+  namespace: gangway
   labels:
     app: gangway
 spec:
   type: LoadBalancer
```

Comment:
This will expose gangway on 8080 from a load balancer. Was the intention to make this a ClusterIP?

Reply:
good catch! I'll fix that.  Also forgot to delete the stuff as I broke them out into different files.